egs/wsj/s5/utils/validate_data_dir.sh

```diff
@@ -126,7 +126,7 @@ fi
 num_utts=`cat $tmpdir/utts | wc -l`
 if ! $no_text; then
   if ! $non_print; then
-    if locale -a | grep "C.UTF-8" >/dev/null; then
+    if locale -a | grep "C.utf8\|C.UTF-8" >/dev/null; then
```

Comment:
Do we know that locales will accept different spellings?
It would be safer to add another clause to the if-statement to set L=C.utf8 if that's what is available.